lib/chef/provisioning/docker_driver/driver.rb

```diff
@@ -134,6 +147,8 @@ def allocate_image(action_handler, image_spec, image_options, machine_spec)
           :from_image => true
```

Comment:
I'm beginning to see some of the confusion around `from_image` - we should rename this in the reference.  Is it even necessary?